Fix: Replaced documentation logo with the new one

Signor1 opened the pull request:
### Description
Updated the Starknet-ID documentation logo as per issue #62 
- Added new logo as `/public/new-logo.svg` (134x28 pixels), preserving the original `/public/logo.svg`.
- Modified `logo: <img/>` in `theme.config.jsx` to use `src="/new-logo.svg"` with `style={{ height: "36px", maxWidth: "172px" }}` and `alt="Starknet-ID"` for accessibility (reduced height from 60px).
- Tested locally on `http://localhost:3000`:
  - Verified logo renders correctly in the header.
  - Confirmed proper placement and responsiveness on mobile and desktop devices.
  - Ensured no console errors or layout issues.
- Ran `npm run build` successfully (see screenshot below).
- No test folder present, but confirmed correct logo display across multiple devices.

### Screenshots
<img width="1280" alt="Screenshot 2025-04-25 at 5 13 31 PM" src="https://github.com/user-attachments/assets/29518d84-8c22-4aa5-88d7-8b558ff7e422" />


### Checklist
- [x] Tested locally and verified build.
- [x] Assigned to issue #62 
- [x] Added new logo without removing the original.
- [ ] Set PR label to "ready for review" (no permission; requested maintainer to apply).
- [x] PR closes issue with `Close #62 `.

Note: I lack permission to set labels. Please apply the “ready for review” label.

Close #62

Signor1 commented:
Hi @Marchand-Nicolas, I’ve submitted the PR and completed local testing and build verification. I don’t have permission to set labels, so please apply the “ready for review” label. Thank you!

Marchand-Nicolas reviewed (changes requested):
Can you delete the old logo as it not used anymore ?
And rename the new one with the same name as the previous one please: logo.svg (don't forget to update the image path)

Signor1 commented:
I've updated it. 

- **Updated per review**: Deleted `/public/logo.svg`, renamed `/public/new-logo.svg` to `/public/logo.svg`, and updated `theme.config.jsx` to `src="/logo.svg"`.

Marchand-Nicolas reviewed (approved):
Great job, lgtm!
